zulip/zulip/__init__.py

```diff
@@ -588,12 +588,16 @@ def do_register():
                 else:
                     res = self.register(event_types=event_types, narrow=narrow)
 
-                if 'error' in res.get('result'):
-                    if self.verbose:
-                        print("Server returned error:\n%s" % res['msg'])
-                    time.sleep(1)
-                else:
-                    return (res['queue_id'], res['last_event_id'])
+                try:
```

Comment:
I think we can make these diffs smaller and the exception handling less aggressive as described here:

https://chat.zulip.org/#narrow/stream/integrations/subject/Strict-optional.20mypy.20.23P250/near/433795